spinn_front_end_common/interface/buffer_management/buffer_manager.py

```diff
@@ -177,6 +177,7 @@ def load_initial_buffers(self):
             total_data, "Loading buffers ({} bytes)".format(total_data))
         for vertex in self._sender_vertices:
             for region in vertex.get_regions():
+                vertex.rewind(region)
```

Comment:
In my implementation, the runtime is reset to 0 after every run. So the correct behaviour for SpikeSourceArrays would be to start over.

If pyNN dictates a different behaviour, this will have to be changed.